### PR TITLE
Harness

### PR DESCRIPTION
**Pull request by nkoenig**
This is a ROS wrapper to [this Gazebo plugin](https://bitbucket.org/osrf/gazebo/pull-requests/2346/added-a-harness-plugin/diff).

### CODE REVIEW

**Comment by nkoenig** [comment]
[DRCSim usage example](https://bitbucket.org/osrf/drcsim/pull-requests/544)

**Comment by hsu** [comment]
reworking [gazebo pr](https://bitbucket.org/osrf/gazebo/pull-requests/2346/added-a-harness-plugin/activity)

**Comment by j-rivero** [comment]
let's wait until the gazebo pull request is reviewed and merged

**Comment by j-rivero** [comment]
@osrf-jenkins run test please

File: gazebo_plugins/CMakeLists.txt [hunk 1]
```diff
@@ -81,6 +81,10 @@ if (GAZEBO_VERSION VERSION_GREATER 6.0 OR GAZEBO_VERSION VERSION_EQUAL 6.0)
   catkin_package(  INCLUDE_DIRS include LIBRARIES gazebo_ros_elevator)
 endif()
 
+if (GAZEBO_VERSION VERSION_GREATER 7.3 OR GAZEBO_VERSION VERSION_EQUAL 7.3)
```
Comment (scpeters) [inline gazebo_plugins/CMakeLists.txt]: I think the following is equivalent logic:

```
if (NOT GAZEBO_VERSION VERSION_LESS 7.3)
```

Comment (scpeters) [inline gazebo_plugins/CMakeLists.txt]: changed in 45b578e

**Comment by scpeters** [comment]
I squashed two commits that were just for fixing the build

**Comment by scpeters** [comment]
+1

**Comment by scpeters** [comment]
let's merge forward or cherry-pick after merging this

**Comment by hsu** [comment]
+1

**Comment by scpeters** [comment]
ok I'll merge

**Comment by scpeters** [comment]
i just cherry-picked forward to jade and kinetic

**Pull request merged**